system/modules/core/classes/Ajax.php

```diff
@@ -195,6 +196,7 @@ public function executePostActions(\DataContainer $dc)
 			exit;
 		}
 
+		/** @var \DataContainer|object $dc */
```

Comment:
As far as I know, `DataContainer` does not accept arbitrary properties, so this should not be `object` but have `@property` for the available options.